BCStudentSoftwareDevTeam/celts
# Change sorting order of events on Events List page.

*There is no Github issue attached to this PR
Requested change: sort events on the Events List page chronologically instead of alphabetically. 

Did: 
- Changed it so that events on the Events List page are sorted chronologically (by start date and start time).
- Moved the "Event Program" column to the first column and renamed it to just "Program".
- Added the order_by of the start time to the Upcoming Events section to a users profile.
- Added more detail on the RSVP'd column of a users Upcoming Events.
- Fixed the test that broke with the new chronological ordering.

To test: 
- With a freshly reset database run the tests.
- After starting the application create multiple events that are apart of the same program and make sure they are sorted by both start date and start time.
- Create a events that require the user to RSVP and check that the RSVP status on the User Profile matches the users RSVP'd status.
